### PR TITLE
Integrate Account Management

### LINKED ISSUE #72: Create Dashboard
**Is your feature request related to a problem? Please describe.**
In order to start integrating the app together, we should add a dashboard for logged in users.

**Describe the solution you'd like**
A dashboard page (partially functioning) that allows users to navigate to other parts of the application.

### Checklist:

- [x] I have double checked that the issue does not currently exist. 
- [x] I have added the appropriate labels (e.g. feature, front-end, back-end).
- [x] I have added the **Platform Project** project

### PR DESCRIPTION
**Pull request by frasermcc9**
# Description

Adds a dashboard, as well as sets up infrastructure for pages in the app.
- useHouse context
- Page layout
- Core navigation

Furthermore, this PR integrates the various parts of this milestone together into the application so they work together. It includes modifications to these PRs to either
 - Fix small issues
 - Improve responsiveness through alerts
 - Replace code structures with first-class supported methods

This also adds Illustrator files for the logo design into git LFS.

## Recommended Testing List
- Create a flat
  - That flat appears in database
  - That flat has you as the owner
  - Your user object contains a reference to that flat
- Open the share modal
  - Copy the link
  - Paste the link into your address bar. Should be warned you can only be in one flat.
  - Sign out (user dropdown).
  - Paste the link into your address bar. Sign in with same account. Should be warned.
  - Sign out
  - Paste the link into your address bar. Sign in with a different account. Should join that flat.
- Go to the `/account` page on a user without a flat (can delete entries in mongodb)
  - Try join a flat using the code of that flat
  - You should be added

## Known Problems
- Phone number is saved into email. We'll have to choose which of these we want.

Fixes/resolves #72 

## Screenshots

Desktop view
image

Mobile view
image

- [X] **New feature** (non-breaking change which adds functionality)

# Checklist:
Leave blank if not applicable

I have completed these steps when making this pull request:
- [X] I have assigned my name to the issue
- [X] I have moved the issue to the **In Progress** column
- [X] I have labelled the PR appropriately
- [X] I have assigned myself to the PR

Before opening the PR for review:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have moved the linked issue to the **Review in Progress** column

### CODE REVIEW

**Review by akash744 — DISMISSED** [review]
Looks Amazing.

**Review by FriedhelmWS — DISMISSED** [review]
Looks nice

**Comment by frasermcc9** [comment]
Sorry I should have had this as draft since I still needed to add link sharing

**Comment by frasermcc9** [comment]
image

Added now after #54 was merged

**Comment by j-chad** [comment]
I'm not a massive fan of the different header colours. I get why they're used to differentiate the house name though. Perhaps a more purpley colour would fit better?
image

**Comment by frasermcc9** [comment]
I agree, maybe just
Welcome to (gray) my house (teal)

then the other heads just remain purely gray

**Comment by alyu201** [comment]
Could we maybe replace the task button with just the icon when a task is completed? Or maybe set the button to light if we're having the button as a toggle? Might be good to help incomplete tasks stand out more
image

**Comment by frasermcc9** [comment]
> Could we maybe replace the task button with just the icon when a task is completed? Or maybe set the button to light if we're having the button as a toggle? Might be good to help incomplete tasks stand out more image

Those are there just for examples. The intent (if this is implemented) is that completed tasks wont show up here at all (they will show up when incomplete, then you can click the thing to finish them, next time the page is loaded it will be gone)

In other places these components are used, we could gray out the top to indicate it is done.

**Comment by alyu201** [comment]
Oh yep I see. In that case, everything looks great :)

File: packages/frontend/package.json [hunk 1]
```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@headlessui/react": "^1.5.0",
```
Comment (yzhe819) [inline packages/frontend/package.json:6]: Do we need headlessui and nextUI both?

Comment (frasermcc9) [reply packages/frontend/package.json:6]: nextui doesnt provide some components (dropdown) whereas headlessui does, and headlessui are fully unstyled meaning we can fully customize them to be in our design style.

Comment (yzhe819) [reply packages/frontend/package.json:6]: Oh, yes. They said their dropdown is coming soon. Just keep headlessUI and nextUI

**Review by Sparye — APPROVED** [review]
Looks nice and responsive on MS Edge in macOS

**Review by yzhe819 — APPROVED** [review]
LGTM

**Review by FriedhelmWS — APPROVED** [review]
LGTM

**Pull request merged**